src/taco-cli/cli/utils/settings.ts

```diff
@@ -87,7 +87,7 @@ class Settings {
             Settings.determinePlatformsFromOptions(options)
         ]).spread<Settings.IPlatformWithLocation[]>(function (supportedPlatforms: CordovaHelper.IDictionary<any>, platforms: Settings.IPlatformWithLocation[]): Settings.IPlatformWithLocation[] {
             var filteredPlatforms = platforms.filter(function (platform: Settings.IPlatformWithLocation): boolean {
-                var supported = !supportedPlatforms || platform.platform in supportedPlatforms;
+                var supported = !supportedPlatforms || platform.platform in supportedPlatforms || platform.location === Settings.BuildLocationType.Remote;
```

Comment:
not sure if I follow this part. why don't we log this warning under determinePlatformsFromOptions? in determinePlatformsFromOptions we are anyways dealing with supported platforms right? so we can say read all remote + local platforms, show a warning if this is not one of the supported ones, attach remote/local and return back IplatformWithLocation[]